webui-src/app/theme.css

```diff
@@ -327,17 +332,18 @@ a.tab-menu-item {
 }
 .tab-menu-item:hover {
   background-color: #999;
+  width: -webkit-fill-available;
 }
+
 .tab-menu-item.selected-tab-item {
   background-color: #3ba4d7;
   color: white;
+  width: -webkit-fill-available;
 }
 
 .sidebar {
-  position: fixed;
-  height: 100%;
-  width: 200px;
-  background-color: white;
```

Comment:
I think the previous styles were better here, fit with the main sidebar quite well:
![0](https://user-images.githubusercontent.com/20013884/117197353-58a1cb80-ae05-11eb-8fe4-6fd7624c7110.png)

Reply:
We don't need the `margin-top` anymore too.

Reply:
> I think the previous styles were better here, fit with the main sidebar quite well:
> ![0](https://user-images.githubusercontent.com/20013884/117197353-58a1cb80-ae05-11eb-8fe4-6fd7624c7110.png)
Sidenavbar looks like this in my browser, why I am getting different styling ?
![Screenshot from 2021-05-06 09-15-46](https://user-images.githubusercontent.com/52042648/117239074-ba852400-ae4b-11eb-8ea4-5c2c77c56d1a.png)

Reply:
I had reverted your changes for the selected section here.

Reply:
okay